pyp2rpm/templates/fedora.spec

```diff
@@ -33,9 +33,9 @@ Summary:        %{summary}
 {{ data.description|truncate(400)|wordwrap }}
 {% endfor -%}
 {%- if data.sphinx_dir %}
-%package -n {{ data.pkg_name|macroed_pkg_name(data.srcname)|name_for_python_version(pv, True) }}-doc
+%package -n {{ data.pkg_name|macroed_pkg_name(data.srcname)|name_for_python_version(None, True) }}-doc
 Summary:        {{ data.name }} documentation
-%description -n {{ data.pkg_name|macroed_pkg_name(data.srcname)|name_for_python_version(pv, True) }}-doc
+%description -n {{ data.pkg_name|macroed_pkg_name(data.srcname)|name_for_python_version(None .base_python_version, True) }}-doc
```

Comment:
This part of template is out of for loop scope, pv is undefined so the version argument used to be an empty string. Passing None to the filter doesn't change the behavior of the template at all.  Only one doc subpackage is created for both versions in case the package includes sphinx documentation and the name of the package is simply python-%{pypi_name}-doc. I am not exactly sure if this violates the last update of Fedora Packaging Guidelines for Python but it is not part of the issue, this PR intends to fix.